lightkurve/search.py

```diff
@@ -441,24 +535,39 @@ def _search_products(target, radius=None, filetype="Lightcurve", cadence='long',
     -------
     SearchResult : :class:`SearchResult` object.
     """
-    observations = _query_mast(target, project=mission, radius=radius)
 
-    # mask out FFIs from observations
-    mask = np.array(['FFI' not in obs['target_name'] and
-                     'FFI' not in obs['obs_collection'] for obs in observations])
+    observations = _query_mast(target, project=mission, radius=radius)
 
-    if len(observations[mask]) == 0:
+    if len(observations) == 0:
         raise SearchError('No data found for target "{}".'.format(target))
 
-    products = Observations.get_product_list(observations[mask])
-    result = join(products, observations[mask], keys="obs_id", join_type='left',
+    products = Observations.get_product_list(observations)
+    result = join(products, observations, keys="obs_id", join_type='left',
                   uniq_col_name='{col_name}{table_name}', table_names=['', '_2'])
     result.sort(['distance', 'obs_id'])
 
-    masked_result = _filter_products(result, filetype=filetype, campaign=campaign,
-                                     quarter=quarter, cadence=cadence,
-                                     project=mission, month=month, sector=sector,
-                                     limit=limit)
+    masked_result = _filter_products(result, filetype=filetype,
+                                     campaign=campaign, quarter=quarter,
+                                     cadence=cadence, project=mission,
+                                     month=month, sector=sector, limit=limit)
+
+    # If Full Frame Images are found, add a table entry for FFI cutouts
+    for i in np.where('TESS FFI' in observations['target_name'])[0]:
+        # if target passed in is a SkyCoord object, convert to RA, dec pair
+        if isinstance(target, SkyCoord):
+            target = '{}, {}'.format(target.ra.deg, target.dec.deg)
+        # pull sector numbers
+        s = observations['sequence_number'][i]
+        # convert to pandas data frame to add FFI cutout row
+        products_df = masked_result.to_pandas()
+        # if the desired sector is available, add a row to the data frame
+        if s in np.atleast_1d(sector) or sector is None:
+            products_df = products_df.append({'description': 'Target pixel file (s{:02})'.format(s),
+                                              'target_name': str(target),
+                                              'productFilename': 'TESScut Full Frame Image Cutout'},
+                                              ignore_index=True)
```

Comment:
I think `products_df` has a `sequence_number` column.  I recommend setting it to `observations['sequence_number'][i]` so that the sector number will easily be available in `SearchResult.table` for cutouts.